modes/settings_menues.h

```diff
@@ -73,43 +73,64 @@ class ShowLengthStyle {
   int len_;
 };
 
+template<class T>
```

Comment:
avoids SHORT_STYLE.
Blade would just light up all LEDs if maxlen was 10 or under.
You likely have a better way of doing this, but I didn't want to also edit show_color.h if that should be avoided?